include/circt/Dialect/FIRRTL/FIRRTLTypes.h

```diff
@@ -68,7 +68,7 @@ class FIRRTLType : public Type {
   /// Reset, etc) then return the bitwidth.  Return -1 if the is one of these
   /// types but without a specified bitwidth.  Return -2 if this isn't a simple
   /// type.
-  int32_t getBitWidthOrSentinel();
+  int32_t getBitWidthOrSentinel() const;
```

Comment:
This shouldn't be marked const, as [MLIR doesn't do this for types](https://mlir.llvm.org/docs/Rationale/UsageOfConst/).